Aztec/Classes/Converters/HTMLNodeToNSAttributedString.swift

```diff
@@ -113,13 +113,12 @@ class HMTLNodeToNSAttributedString: SafeConverter {
         let childAttributes = attributes(forNode: node, inheritingAttributes: inheritedAttributes)
 
         for child in node.children {
-            let childContent = convert(child, inheritingAttributes: childAttributes)
-
+            let childContent = NSMutableAttributedString(attributedString:convert(child, inheritingAttributes: childAttributes))
             content.appendAttributedString(childContent)
         }
 
         if node.isBlockLevelElement() && !node.isLastChildBlockLevelElement() {
-            content.appendAttributedString(NSMutableAttributedString(string: "\n", attributes: childAttributes))
+            content.appendAttributedString(NSMutableAttributedString(string: "\n", attributes: inheritedAttributes))
```

Comment:
Following up on the logic above, this can be safely changed to `NSAttributedString`.